edx_sga/tasks.py

```diff
@@ -66,20 +76,55 @@ def _collect_student_submissions(block_id, course_id, locator, destination_path)
                 log.exception("Unable to download submission of student=%s", student.username)
 
 
-def _compress_folder(destination_path):
+def _get_destination_file_path(student_name, sha1, ext, destination_path):
     """
-    compress given folder/file
+    return destination file path
 
     Args:
+        student_name (str): user name of student
+        sha1 (str): SHA code for file
+        ext (str): extension of file
         destination_path (str): path (including name) of folder/file which we want to compress.
     """
-    make_archive(
-        destination_path,
-        'zip',
-        destination_path
+    destination_file_name = six.u("{student_name}_{sha1}{ext}").format(
+        student_name=student_name,
+        sha1=sha1,
+        ext=ext
+    )
+    return os.path.join(
+        destination_path, destination_file_name
     )
 
 
+def _compress_folder(destination_path, zip_file_path):
+    """
+    compress given folder/file
+
+    Args:
+        destination_path (str): path (including name) of folder/file which we want to compress.
+    """
+    from edx_sga.sga import BLOCK_SIZE
```

Comment:
If possible, please move `BLOCK_SIZE` to `constants.py` and import at the module level instead of importing within this function